basement_bot/plugins/google.py

```diff
@@ -13,7 +13,13 @@ def get_items(url, data):
     return requests.get(url, params=data).json().get("items")
 
 
-@commands.command(name="g")
+@commands.command(
+    name="g",
+    brief="Googles that for you",
+    description="Returns the top Google search result of the given search terms. Returns nothing if one is not found.",
+    usage="[search-terms]",
+    help="\nLimitations:\n    Mentions should not be used.",
```

Comment:
It would be nice to make a helper function that formats these line breaks for you. Seems useful for someone adding a new plugin.

Reply:
I'd have to figure that out in another PR.